app/views/shared/_setlist_card.html.erb

```diff
@@ -4,6 +4,8 @@
       <div class="imgBx">
         <% if setlist.setlist_image.attached? %>
           <%= cl_image_tag setlist.setlist_image.key %>
+        <% elsif setlist.tracks.empty? %>
+          <%= cl_image_tag 'app/assets/images/sets_logo.svg' %>
```

Comment:
This doesnt always work, @scallywag89 You may have to look over this one

Reply:
Will have a look, think I might see the issue there

Reply:
@ninjacazaam firstly, I think that we don't really need that elsif - if there is no user image i think it's cleaner to just use the first album cover. if we do want the elsif then it should be an image_tag not a cl_image_tag i'm pretty sure!

Reply:
If there is no user image and there are no tracks in the setlist then it throws an error, hence why this elsif exists